src/Resources/config/routing.yml

```diff
@@ -35,6 +35,16 @@ bitbag_sylius_wishlist_plugin_shop_wishlist_add_selected_products:
     defaults:
         _controller: bitbag_sylius_wishlist_plugin.controller.action.add_selected_products_to_cart
 
+bitbag_sylius_wishlist_plugin_shop_wishlsit_export_selected_products_to_csv:
+    path: /wishlist/export/csv
```

Comment:
Reverse order: `/wishlist/csv/export|import`; think of domain names: their structure comes from wide context to more and more narrow.